src/jdk.accessibility/windows/native/bridge/AccessBridgeCalls.c

```diff
@@ -230,6 +230,7 @@ extern "C" {
             }
             return TRUE;
         }
+        ((void) error);
```

Comment:
Not sure why do we need it. I mean there is a value assignment before so why would compiler complain?

Reply:
The error being cast to void is just there to mark that this was a place that the compiler flagged, as for the warning itself, apparently just assigning the value is not enough, as that yields a \-Wunused\-but\-set\-variable, it has to actually be used somehow or explicitly be cast to void to stop the warning altogether

Reply:
This may be useful in a debug build to look up the error if it occurs…

Shall it be removed?